src/ring_theory/dedekind_domain/adic_valuation.lean

```diff
@@ -331,4 +331,100 @@ instance adic_completion.has_lift_t : has_lift_t K (v.adic_completion K) :=
 /-- The ring of integers of `adic_completion`. -/
 def adic_completion_integers : valuation_subring (v.adic_completion K) := valued.v.valuation_subring
 
+instance : inhabited (adic_completion_integers K v) := ⟨0⟩
+
+variables (R K)
+
+lemma adic_completion.is_integer (x : v.adic_completion K) :
+  x ∈ v.adic_completion_integers K ↔ (valued.v x : ℤₘ₀) ≤ 1 :=
+iff.rfl
+
+section algebra_instances
+
+/- The next two lemmas cannot be made instances, since the uniform space structure on `K`
```

Comment:
Why does this matter? This instance looks safe to me

Reply:
Eric is right: these can indeed be instances.

Checking now, I seem to have been [the source](https://github.com/leanprover-community/mathlib/pull/14249#discussion_r999241935) of this misinformation. I cannot understand my original comment at all: the target type obviously contains `v` and is not even similar to `algebra R K`! Sorry for the confusion.

Reply:
Yes, that comment was the reason for this change. I've changed it back to the original instance.